.github/CODEOWNERS

```diff
@@ -913,7 +913,7 @@
 /**/*Management*/                                                  @archerzz @ArcturusZhang @ArthurMa1978
 
 # PRLabel: %Mgmt
-/**/Azure.ResourceManager*/                                        @archerzz @ArcturusZhang @ArthurMa1978
+/**/Azure.ResourceManager*/                                        @archerzz @ArcturusZhang @ArthurMa1978 @bicep-admins
```

Comment:
You do not want to do this.   That will include your team for ALL resource manager libraries in the repository.